charts/tileserver-importer/templates/job.yaml

```
@@ -0,0 +1,47 @@
+---
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name:  {{ include "tileserver.name" . }}
+  labels:
+    {{- include "importer.labels" . | nindent 4 }}
+spec:
+  backoffLimit: 0
+  template:
+    metadata:
+      name: {{ include "tileserver.name" . }}
+    spec:
+      serviceAccountName: {{ include "tileserver.name" . }}
+
+      volumes:
+        - name: tmp-volume
+          emptyDir: {}
+
+        - name: config-volume
+          configMap:
+            name: {{ include "tileserver.name" . }}
+
+      {{- with .Values.tileserver.importer.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+
+      containers:
+        - name: import-worker
+          {{- with .Values.tileserver.importer }}
+          image: {{ .image }}:{{ .tag }}
+          {{- end}}
+          command: ["/selfimporter", "scheduler"]
```

Стоит добавить `resources`

```suggestion
          command: ["/selfimporter", "scheduler"]
          {{- with .Values.tileserver.importer.resources }}
          resources:
            {{- toYaml . | nindent 12 }}
          {{- end }}
```